# Will be encapsulated after the ros node receiver is added and working
std_msgs/Header header
bool is_valid
int64 count
int64 struct_id
int64 version
int64 system_type
int64 system_subtype
int64 fw_major_version
int64 fw_minor_version
int64 fw_patch_version
int64 fw_build_version
int64 year
int64 month
int64 day
int64 hour
int64 minute
int64 second
int64 coordinate_system
# vector3 -> velocity
float64 vel_x
float64 vel_y
float64 vel_z
geometry_msgs/Vector3 displacement
float64 vel_error
float64 range_beam1
float64 range_beam2
float64 range_beam3
float64 range_beam4
float64 mean_range
float64 speed_of_sound
int64 status
int64 bit_count
int64 bit_code
float64 voltage
float64 transit_voltage
float64 current
std_msgs/String serial_number